packages/webpack-loader/src/index.ts

```diff
@@ -33,7 +33,7 @@ function removeDescriptions(doc: DocumentNode) {
   return doc;
 }
 
-interface Options {
+export interface Options {
```

Comment:
I think this one could be seperated PR for CI.

Other part looks good. just update versions.